js/apps/account-ui/playwright.config.ts

```diff
@@ -9,7 +9,7 @@ export default defineConfig({
   forbidOnly: !!process.env.CI,
   retries: process.env.CI ? 2 : 0,
   workers: 1,
-  reporter: "html",
+  reporter: process.env.CI ? "github" : "list",
```

Comment:
I've set the default reporter to `list`, as for local development one would already be using the UI-mode.